network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/typeform_block.html

```diff
@@ -0,0 +1,21 @@
+<script
+  src="https://embed.typeform.com/embed.js"
+  type="text/javascript"
+></script>
+
+<div
+  id="typeform-embed"
+  class="typeform-embed"
+  style="height: {{ value.height }}px;"
+></div>
+
+<script type="text/javascript">
```

Comment:
Let's make this code that gets invoked in `source/js/main.js` (but probably housed in its own file) - we already do this for a fair number of other things, so there should be some examples to copy from, and also means we're not unsafe-inlining our own code.

Reply:
How would we get the form URL passed in? look for a meta tag?